### ISSUE TITLE
Recipes that require Klein Star Omega do not require it to be full

### ISSUE THREAD

**Issue by psychofad** [sinkillerj/ProjectE#359]
In keeping with the idea that this is a rewrite of EE2, the Gem Armor and Alchemical Tome recipes should require full Klein Star Omegas to craft.  This is not the case currently as an empty klein star may be used to craft these items.  In my private server pack, I have used minetweaker and tweaked the recipe with "<ProjectE:item.pe_klein_star:5>.onlyWithTag({StoredEMC: "51200000"})" forcing it to only accept the full Klein Star, but this is really just a bandaid and doesn't render the full item in the recipe in NEI.

This will noticeably increase the cost of the armor and tome EMC-wise so if the recipe is adjusted then the EMC for the items would also have to be increased to take into account for the Stored EMC in the Klein Stars.  At this time it appears that adding EMC to a klein star of any size does not increase it's worth at all and if someone was to put it in the burn slot of the Transmutation Tablet all of the stored EMC would be lost due to this.  Not sure if that issue has ever been brought up, but if not I would be happy to make a separate issue on this GitHub in regards to this.  Just let me know if this needs to be done.

**Comment by sinkillerj** [comment]
One stored EMC issue was solved in a merge today, ill look into the Klein Stars though cause I dont think they are fixed by it.

As for the full Klein Star, I kinda feel like that would add unnecessary confusion. I may change this, but only once a help guide is implemented.

**Comment by psychofad** [comment]
I can understand the confusion to players who have never used EE2.  Just seems like the extremely OP Gem Armor is a bit too easily obtained.  For reference to it's EE2 counterpart and the use of Full Klein Stars my information was obtained here:
http://tekkitclassic.wikia.com/wiki/Gem_Armor

I realize that this is a spiritual remake as quite a few things deviate from the original mod..I mean...we never had NEI Integration back then, but I just wanted to point this out as it seemed like something that was simply overlooked.  Thanks for looking into the Stored EMC issue.  As for a help guide, for the most part this mod holds to it's spiritual successor and there are plenty of wiki's out there that can help a new user in the interim.

**Comment by psychofad** [comment]
closing this since it was fixed ages ago :P

**Comment by MaPePeR** [comment]
I think this bug reappeared in 1.10, though....

**Comment by williewillus** [comment]
We never fixed it 0.o

**Comment by psychofad** [comment]
ah, my bad, I read the above closed issue and thought it was fixed

**Comment by sinkillerj** [comment]
Truth be told we could easily fix it at any time, but I still consider the change to be unnecessary, potentially causing more harm than good.

**Comment by williewillus** [comment]
i mean the gem armors are extremely powerful, so it makes sense to require full kleins for them

**Comment by MaPePeR** [comment]
It's nearly impossible to fill a Klein star in a reasonable time without a table. And some people try to avoid using  it.

Or is there another fast way I don't know about?

**Comment by williewillus** [comment]
You charge them by burning stuff into a relay

**Comment by MaPePeR** [comment]
Yea, but the relay only transfers so much per tick, I think. And it would take a long time to charge a omega that way.

**Comment by psychofad** [comment]
That was the whole point of requiring them to be full in the original EE2.  It takes a long time, or you have to burn a lot of items.  The armor was SUPPOSED to be hard to get!

**Comment by psychofad** [comment]
And the easiest/fastest way to get them full was to burn items in the transmutation table/tablet and suck all the EMC out of there into the klein star.  A few stacks of red matter ought to do it!

**Comment by MaPePeR** [comment]
With the tablet its pretty much instant - and it does not change the amount of emc you need in total.

**Comment by CrankySupertoon01** [comment]
Still exists in current build. Do klein stars has nbt so i could use craft tweaker to fix it. Thanks

**Comment by psychofad** [comment]
@CrankySupertoon01 see the first post on how I was able to use Minetweaker to do this, CraftTweaker uses the same format as it is a successor to Minetweaker.  Best of luck!
